### PR TITLE
plexamp: 4.8.3 -> 4.10.1

### PR DESCRIPTION
**Pull request by Redhawk18**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Redhawk18** [comment]
Supersedes #313297

File: pkgs/applications/audio/plexamp/default.nix
Comment (Aleksanaa) [inline pkgs/applications/audio/plexamp/default.nix:23]:
```suggestion
      $out/share/icons/hicolor/scalable/apps/plexamp.svg
```

Comment (Redhawk18) [reply pkgs/applications/audio/plexamp/default.nix:23]: Was there any testing on this? why would you make this change, if this is wrong it will cause the package to fail to build.

Comment (Aleksanaa) [reply pkgs/applications/audio/plexamp/default.nix:23]: Please read https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html

**Review by r-burns — APPROVED** [review]
LGTM. Please squash to two commits:

`maintainers: add redhawk`
and
`plexamp: 4.8.3 -> 4.10.1`

and I'll go ahead and merge :+1:

**Comment by Redhawk18** [comment]
> LGTM. Please squash to two commits:
> 
> `maintainers: add redhawk` and `plexamp: 4.8.3 -> 4.10.1`
> 
> and I'll go ahead and merge 👍

I was told directly to make those two different commits.

**Comment by r-burns** [comment]
Yep that's correct, I'm just saying to squash all the other changes to the package's derivation into the version bump commit.

**Review by genusistimelord — APPROVED** [review]
All these changes seem to fit the requirements. So looks good.

**Review by Aleksanaa — APPROVED** [review]

**Comment by Aleksanaa** [comment]
Still needs to squash commits

**Comment by Redhawk18** [comment]
> LGTM. Please squash to two commits:
> 
> `maintainers: add redhawk` and `plexamp: 4.8.3 -> 4.10.1`
> 
> and I'll go ahead and merge 👍

I'm ready

**Review by genusistimelord — APPROVED** [review]
I see the same changes and with the commits squashed as asked for.

**Comment by itslychee** [comment]
Added the label for @Redhawk18 due to them not receiving an org invite after the PR was merged.

**Pull request merged**
